snippets/snippets.code-snippets

```diff
@@ -246,5 +246,60 @@
             "else:",
             "    $0"
         ]
-    }
+    },
+    
+    "discord.py Exception":{
+		"prefix": "!excpt",
```

Comment:
```suggestion
		"prefix": "!except",
```

I don't really like the one letter shortening, something like exc would have been nicer but people will most likely not need to write the full except since vscode has autocompletion